l2geth/README.md

```diff
@@ -16,7 +16,7 @@ configuration will determine the mode of operation. The configuration flags
 can be configured using either environment variables or passed at runtime as
 flags.
 
-A prebuilt Docker image is available at `ethereumoptimism-optimism/optimism/l2geth`.
+A prebuilt Docker image is available at `ethereumoptimism/go-ethereum`.
```

Comment:
should we swap this for `bobanetwork/l2geth`?

Reply:
Please think about the best way to do this, longer term - not sure